src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -156,6 +156,9 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						if (!value.isEmpty() && media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.H264)) {
 							media.setAvcLevel(getAvcLevel(value));
 						}
+						if (isNotBlank(value) && media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.H265)) {
```

Comment:
Oh, you mean all the lock part that i copy past ?
I don't really know, as i just copy past and adapt, but you should be right.
But i'm not sure i'll make thing worst if i try to delete some part.